src/datasets/meshes/faust.jl

```diff
@@ -0,0 +1,201 @@
+│ /home/user/.julia/juliaup/julia-1.7.3+0.x86/local/share/julia/datadeps/MPI-FAUST,
+│ /home/user/.julia/juliaup/julia-1.7.3+0.x86/share/julia/datadeps/MPI-FAUST,
+│ /home/user/datadeps/MPI-FAUST,
+│ /scratch/datadeps/MPI-FAUST,
+│ /staging/datadeps/MPI-FAUST,
+│ /usr/share/datadeps/MPI-FAUST,
+└ or /usr/local/share/datadeps/MPI-FAUST
+[ Info: by following the instructions:
+┌ Info: Dataset: MPI-FAUST.
+└ Website: http://faust.is.tue.mpg.de/
+Once installed please enter 'y' reattempt loading, or 'a' to abort
+[y/a]
+```
+Now download and extract the dataset into one of the given locations. For unix link systems, an example command can be
+```bash
+unzip -q <path-to-filename</filename.zip ~/.julia/datadeps
+```
+The corresponding folder tree should look like
+```
+├── test
+│   ├── challenge_pairs
+│   └── scans
+└── training
+    ├── ground_truth_vertices
+    ├── registrations
+    └── scans
+```
+Press `y` to re-attept loading.
+```julia-repl
+dataset FAUST:
+  scans          =>    100-element Vector{Any}
+  registrations  =>    100-element Vector{Any}
+  labels         =>    100-element Vector{Vector{Bool}}
+  metadata       =>    Dict{String, Any} with 0 entries
+```
+
+## Load train and test split
+
+```julia-repl
+julia> train_faust = FAUST(:train)
+dataset FAUST:
+  scans          =>    100-element Vector{Any}
+  registrations  =>    100-element Vector{Any}
+  labels         =>    100-element Vector{Vector{Bool}}
+  metadata       =>    Dict{String, Any} with 0 entries
+julia> test_faust = FAUST(:test)
+dataset FAUST:
+  scans          =>    100-element Vector{Any}
+  registrations  =>    100-element Vector{Any}
+  labels         =>    100-element Vector{Vector{Bool}}
+  metadata       =>    Dict{String, Any} with 0 entries
+```
+
+## Scan, registrations and ground-truth
+
+```julia-repl
+julia> dataset = FAUST(); # defaults to train split
+julia> scan = dataset.scans[1] # pick one scan
+Mesh{3, Float32, Triangle}:
+ Triangle(Float32[-0.0045452323, 0.08537669, 0.22134435], Float32[-0.0030340434, 0.08542955, 0.22206494], Float32[-0.0042151767, 0.08697654, 0.22171047])
+ Triangle(Float32[-0.05358432, 0.08490027, 0.17748278], Float32[-0.05379858, 0.083174236, 0.17670263], Float32[-0.052645437, 0.08346437, 0.17816517])
+.
+.
+.
+ Triangle(Float32[-0.07851, -1.0956081, 0.07093428], Float32[-0.06905176, -1.0986279, 0.07775441], Float32[-0.069199145, -1.0928112, 0.06812464])
+julia> registration = dataset.registrations[1] # The corresponding registration
+Mesh{3, Float32, Triangle}:
+ Triangle(Float32[0.12491254, 0.51199615, 0.29041073], Float32[0.11376736, 0.5156298, 0.3007352], Float32[0.119374536, 0.50043654, 0.29687837])
+ Triangle(Float32[0.119374536, 0.50043654, 0.29687837], Float32[0.11376736, 0.5156298, 0.3007352], Float32[0.10888693, 0.5008964, 0.30557302])
+.
+.
+.
+ Triangle(Float32[0.033744745, 0.030968456, 0.2359996], Float32[0.058017172, 0.044458304, 0.23422624], Float32[0.03615713, 0.04858183, 0.23596591])
```

Comment:
```suggestion
 Triangle(Float32[0.033744745, 0.030968456, 0.2359996], Float32[0.058017172, 0.044458304, 0.23422624], Float32[0.03615713, 0.04858183, 0.23596591])
 
```

Reply:
I don't see any changes, is there any spacing-related issue?

Reply:
yes, the change is adding a new line